iwyu_preprocessor.cc

```diff
@@ -1087,4 +1129,25 @@ bool IwyuPreprocessorInfo::ForwardDeclareIsInhibited(
       ContainsKey(*inhibited_forward_declares, normalized_symbol_name);
 }
 
+bool IwyuPreprocessorInfo::ForwardDeclareInKeepRange(
+    const SourceLocation& loc) const {
+  // Check if the entry exists, shortcircuit if not
```

Comment:
The style for IWYU comments follows Google style: https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar

Please make them full sentences with capitalization and punctuation. Here and below.